src/Transport/Creation/AzureServiceBusForwardingSubscriptionCreator.cs

```diff
@@ -135,6 +135,46 @@ public AzureServiceBusForwardingSubscriptionCreator(ReadOnlySettings settings)
 
         }
 
+        public async Task DeleteSubscription(string topicPath, string subscriptionName, SubscriptionMetadata metadata, string sqlFilter, INamespaceManager namespaceManager, string forwardTo)
```

Comment:
Why is the delete subscription contained in this PR? I suppose those changes are unrelated to this PR?